### DISCUSSION TITLE
Option to hide crosswalks

### DISCUSSION THREAD

**Discussion by pkubowicz** [osmandapp/OsmAnd#11583] [Maps Related]
# 🚀 feature request

### Description

When I was viewing the map in bright sun, crosswalks looked like arrows of one-way streets. Both have the same colour. In places where there are many crosswalks the map can get very noisy and unreadable. Street name and one-way indicator are critical for navigation. On the other hand, information about crosswalks is of zero interest if you drive – you don't need to plan ahead for them.

See the screenshot:

![crosswalk](https://user-images.githubusercontent.com/13313417/116656140-7813a100-a98c-11eb-8f80-54006918ab63.png)

### Describe the solution you'd like

[Map viewing](https://osmand.net/features/map-viewing) -> Hide should contain an option to hide crosswalks.

The default profile for bicycle should have this option enabled, so that you don't see crosswalks on map.

### Describe alternatives you've considered

Never render crosswalks. Rely on voice prompts to warn about crosswalks.

**Comment by vshcherb** [comment]
Probably it could customized to draw them only on high detailed zooms only. So it won't clutter the map early

**Reply by sonora** [reply]
I agree, that's the best solution,  rather than another settings switch where noone would know in advance anyway in which area they would need to turn it on or off (and foreseeably few users would care to repeatedly toggle such a switch).
